src/Config/ObjectModel/EntityActionOperation.cs

```diff
@@ -19,7 +19,7 @@ public enum EntityActionOperation
     Delete, Read,
 
     // cosmosdb_nosql operations
-    Upsert, Create,
+    Upsert, Create, Patch,
```

Comment:
If we dont need a separate permission for Patch, do we still need a separate operation in EntityActionOperation?

Reply:
Yes, `Patch` is a separate operation and it should be reflected in the DAB code.

Reply:
I think this represents high level of operation type needed for permissions versus MutationEngine code path execution:

| GraphQL Mutation | Permission    | DAB OperationType / CosmosMutationEngine code path |
|------------------|---------------|----------------------------------------------------|
| patchEntityByPk  | Update        | Patch                                              |
| updateEntityByPk | Create,Update | Update                                             |